app.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'lib/manage_books'
+require_relative 'lib/manage_music'
+require_relative 'lib/manage_games'
+
+class App
+  attr_accessor :books, :music, :games
+
+  def initialize
+    @books = []
+    @music = []
+    @games = []
+    @labels = []
+    @authors = []
+    @genres = []
+  end
+
+  def load_data
+    puts 'Loading data here..'
+    load_books
+    load_game_data
+    saved_albums
+  end
+
+  def save_exit
+    puts 'Goodbye'
+
+    # save_books
+
+    # save_music
+
+    # save_games
+
+    exit
+  end
+
+  def invalid_option
+    puts 'Invalid option'
+  end
+
+  def options
+    puts 'Please enter the number for the task you want to perform'
+    puts 'choose one of the following'
+    puts '1 - Manage books.'
+    puts '2 - Manage Music.'
+    puts '3 - Manage Games.'
+    puts '4 - Exit'
+  end
```

Comment:
- [ ] Good job creating a very UI management for you app 👏 , I really liked the experience you provided for the users 👏 👏 👏. As describe in the UML `Book`, `Music` and `Game` table inherit from the `Item` table , Means that each **Item** should have a `author` , `genre` and a `label`. On your side you implemented this partially. Because only `books` have `labels` , only `Musics` have `genres` and only `games` have `authors` , Kindly fixed that 

| Book | Music | Game |
| :----: | :---: | :----: |
| ![Capture](https://github.com/abeltsew/ruby_capstone/assets/80458199/06d99281-9fe9-4515-8aef-963696cfd1d3) | ![Capture1](https://github.com/abeltsew/ruby_capstone/assets/80458199/a5357e38-f6e1-4a5c-a709-4f1cf1d60771) | ![Capture2](https://github.com/abeltsew/ruby_capstone/assets/80458199/3de332c7-c86a-48dc-84ec-00e296b046ec) |